content/docs/start/experiments.md

```diff
@@ -21,39 +21,34 @@ the [`example-dvc-experiments`][ede] project.
 
 <details>
 
-### ⚙️ Installing the example project
+### ⚙️ Initializing a project into DVC experiments
```

Comment:
into -> to use?

(it's easier to read I think)

Reply:
Updated in 5451d8227